Model/PayPalManagement.php

```diff
@@ -198,14 +198,17 @@ protected function _buildCheckoutRequest($payment,$deviceFingerprint)
         $request['ReferenceId'] = $order->getIncrementId();
         $request['ConsumerCurrency']= $order->getOrderCurrencyCode();
         $request['DeviceFingerprint'] = $deviceFingerprint;
-        $order->getOrderCurrencyCode();
+        $order->getOrderCurrencyCode(); //what it is for?
```

Comment:
this looks like nonsense, did you try commenting it out? The same call is used a couple of lines above to set the ConsumerCurrency which makes sense, but this call looks superfluous

Reply:
you know what, I wonder if originally this call was set to a variable that could be used in all of the convertCurrency calls below and they just took out the variable part?

Reply:
Honestly I am not sure why it is that way. I wanted to add the comment by side to start a conversation on it during review.

Reply:
It feels like a typo to me. Removing it (unless there is an objection). 
Commit: ffe3f6442e04e765e9356cdc9e60f93815dffa0a
Thanks.